bot.py

```diff
@@ -111,10 +115,11 @@ def maxi(white_team: teams.Team, bot_team: teams.Team, depth, min_from_previous_
                     best_move = (piece, move_square)
                     best_score = score_after_move
 
-
             except chess_utils.DidntMove:
                 pass
 
+    if best_score == MINIMUM_SCORE:
```

Comment:
Add comment before and explain what are you doing here.

Reply:
:eagle: